cookbooks/letsencrypt/templates/default/request.erb

```diff
@@ -2,6 +2,13 @@
 
 # DO NOT EDIT - This file is being maintained by Chef
 
+if [ "$(id -un)" != "letsencrypt" ] || [ "$(id -gn)" != "letsencrypt" ]; then
+  echo "Error: must be running as user/group 'letsencrypt'"
+  exit 1
+fi
+
+cd /srv/acme.openstreetmap.org
```

Comment:
Why do you think this is needed? Does this script not work regardless of what directory you're in? It uses absolute paths for everything?